skyportal/tests/api/test_annotation_services.py

```diff
@@ -90,3 +90,42 @@ def test_vizier_quasar(public_group, upload_data_token):
             for d in data['data']
         ]
     )
+
+
+@pytest.mark.flaky(reruns=2)
```

Comment:
Do we know why this is flaky (if at all?)

Reply:
I mark them as such as it is an external query, we could maybe remove but it seems better to give it a few tries.